.gitignore

```diff
@@ -4,4 +4,3 @@ npm-debug.log.*
 htmldocs/
 lerna-debug.log
 lib/
-.DS_Store
```

Comment:
Put it another way—you shouldn't ever really commit `*.logs` or the `node_modules` folder, yet these files/folders are _always_ added to a project's `.gitignore` as opposed to assuming that they're in a user's global `.gitignore`.

Why should `node_modules` and `.log` files be treated any differently to system files like `.DS_Store`? 

Having some files/folders ignored globally assumes that everyone else that may work on the repo has the same global settings as you.